zebra/zebra_rib.c

```diff
@@ -1099,34 +1110,48 @@ static void rib_process(struct route_node *rn)
 		 * is not caught in the nexthop_active_update() code. Thus, we
 		 * might miss changes to recursive NHs.
 		 */
```

Comment:
not your fault, but the comment block up here is a bit confusing - I don't think it's kept up with the evolution of the selection code. it implies that we're going to call nexthop_active_update() more than once, and that recursive resolution happens in some other path.

Reply:
fixed the comment